Hi3Helper.Core/Lang/en-us.json

```diff
@@ -820,6 +820,8 @@
     "Graphics_SubsurfaceScattering": "Subsurface Scattering",
     "Graphics_TeammateFX": "Teammate Effects",
     "Graphics_AnisotropicFiltering": "Anisotropic Filtering",
+    "Graphics_DisableTeamPageBackground": "Disable Team Page Background",
```

Comment:
Would need to check to see if the setting name matches the one in-game as we don't want to confuse the user.

Reply:
Adjusted as of https://github.com/neon-nyan/Collapse/pull/231/commits/a3f61ff4c582905d1824c7774ab55c8dc8658360